a11ym

```diff
@@ -250,4 +263,43 @@ crawler.on('fetchcomplete', function(queueItem) {
     urls.push({url: url});
 });
 
-crawler.start();
+var computeURLToCrawler = function (crawler) {
+    var first = true;
+
+    return function (value) {
+        var url = crawler.processURL(value);
+        console.log(url);
+
+        if (true === first) {
+            first                   = false;
+            crawler.host            = url.host;
+            crawler.initialProtocol = url.protocol;
+            crawler.initialPort     = url.port;
+            crawler.initialPath     = url.path;
+        } else {
+            ++maximumUrls;
+        }
+
+        crawler.queue.add(
+            url.protocol,
+            url.host,
+            url.port,
+            url.path
+        );
+    };
+};
+
+if (1 === program.args.length && '-' === program.args[0]) {
+    crawler.maxDepth = 1;
```

Comment:
Wait and see yes. I did this because:
  * It makes sense, when you specify a list of URL, this is what you would like to test, not to crawl,
  * It was introducing a big mess with the maximum URLs counter. Some crawled URLs were computed before the ones in the user list (I didn't understand why, the crawler queue behaves strangely sometimes…), and then the maximum URLs counter were reached before having computed all the URL in the list.

So, wait and see :-p.